cards/document-standard/template.hbs

```diff
@@ -9,9 +9,9 @@
       </div>
       <div class="HitchhikerCard-actions">
```

Comment:
I had added the `HitchhikerCard-actions` div to have the feedback buttons follow the placement of the CTAs. Since this is no longer wanted, the div and it's styling can probably be removed from all templates

Reply:
got it, will do!

Reply:
I removed all `HichhikerCard-actions`